fix: added computation for weight percentage, reputation, and how muc…

janchris80 opened the pull request:
…h amount user can get

aaashir committed:
release 3.64.0-dev (#1206)

aaashir committed:
release 3.64.1 (with fixes) (#1207)

aaashir committed:
hotfix: buzz button widget and profile pic issue

aaashir committed:
release 3.64.4-dev

aaashir committed:
hotfix: netlify route error

aaashir committed:
hotfix: fixes and updates

aaashir committed:
hotfix: 3.65.0-dev

janchris80 committed:
fix: added computation for weight percentage, reputation, and how much amount user can get

aaashir commented:
![image](https://github.com/d-buzz/d.buzz-client/assets/70917922/de1f91a4-b66e-4d6b-a8bc-69f1c2cc292e)

- It should show the correct amount instead of NaN
- The background theme should match the activated theme

janchris80 committed:
fix: isNaN result and theme

added api fetchActiveVotes
added actions for fetching active votes
added sagas for fetching active votes

janchris80 commented:
@aaashir new code pushed for the fixed of NaN and theme.

![image](https://github.com/d-buzz/d.buzz-client/assets/34290503/007dbe10-e3d7-4f91-b602-967eeaf47e61)
